src/nav/NavBarBackButton.js

```diff
@@ -21,10 +21,11 @@ import * as NavigationService from './NavigationService';
 // TODO: on iOS, give the right label for a back button
 export default function NavBarBackButton(props: {| +color?: string |}) {
   const { color } = props;
+  const accessibilityLabel = Platform.OS === 'ios' ? 'Back Button' : 'Navigate up';
```

Comment:
Let's make sure this `accessibilityLabel` gets translated.

`NavBarBackButton` is a function component that's set up to use [React Hooks](https://reactjs.org/docs/hooks-overview.html); we've recently started converting more of our components into this form. This looks like one of the first components that started its life as a function component (that was in 4e524e89e2182bfc6774d3649d757558e656ec05). There have also been several commits, like dd7cb8866dcb4a87bdf5c33400c57fbc57d59158, that convert a class component to a function component, to prepare for using Hooks.

A consequence of this being a Hooks-based component is that there's a different (and easier!) way to use React Context. Instead of doing `static contextType`, etc., you'll want to do this:

```js
import React, { useContext } from 'react';

// …

export default function NavBarBackButton(props: {| +color?: string |}) {
  // …
  const _ = useContext(TranslationContext`);
  // …

```

Reply:
Also, the "Button" in "Back Button" seems redundant. If it's not already clear that it's a button, that seems like something that can be fixed with an `accessibilityRole` somewhere. Perhaps `Touchable` can accept an `accessibilityRole` prop, and it could have 'button' as the default value, using [`static defaultProps = { … }`](https://reactjs.org/docs/react-component.html#defaultprops)?